PlayerScripts/PlayerScriptsPlace/StarterPlayer/StarterPlayerScripts/ControlScript/MasterControl/TouchJump.rbxmx

```diff
@@ -14,6 +14,9 @@
 local Players = game:GetService('Players')
 local GuiService = game:GetService('GuiService')
 
+local jumpButtonChangeFlagSuccess, jumpButtonChangeFlagValue = pcall(function() return UserSettings():IsUserFeatureEnabled("JumpButtonPositionChange") end)
```

Comment:
Essentially yes, but the flag has to start with the word "User", so just change that flag to UserJumpButtonPositionChange